### DISCUSSION TITLE
Search for Events

### DISCUSSION THREAD

**Discussion by HerrLevin** [Traewelling/traewelling#2004] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
Currently it's Pride season. That's why we're getting immense amounts of events/event suggestions.
During checkin, the dropdown is not rarely longer than the screen of the user which can get very confusing.

**Describe the solution you'd like**
I'd like to have a searchable event dropdown in the checkin process, to be able to filter between many events

**Additional context**
![image](https://github.com/Traewelling/traewelling/assets/1267894/541cfa5a-6fb2-4899-95ca-c55c3d1e7cbc)

**Comment by pascalwittler** [comment]
A search could be a viable option, but I'm not sure more controls and more used space on small screens improves clarity and usability. In addition, I often come across the name of an event by reading the drop-down menu entries, so I probably wouldn't be able to search for it properly.

I also think that the look and feel of `<select>` elements controlled by the web browser and operating system makes more sense than the individually implemented drop-down menus shown in the screenshot. For example, Android already has a well-known, scrollable and touch-optimized solution for more select options than fit on the screen.

However, I wonder if a uniform naming scheme for events (e.g. all Christopher Street Day events starting with "CSD" followed by the city) and alphabetical sorting would not already provide more clarity.
